src/Microsoft.DotNet.Darc/DarcLib/Models/VirtualMonoRepo/SourceManifest.cs

```diff
@@ -110,13 +110,10 @@ public string ToJson()
             WriteIndented = true,
         };
 
-        var data = new SourceManifestWrapper
-        {
-            Repositories = _repositories,
-            Submodules = _submodules,
-        };
+        // Wrap SourceManifest for serialization
+        var sourceManifestWrapper = ToWrapper();
```

Comment:
This is my personal opinion, but the method call looks a bit weird like this. I think I'd much prefer if `ToWrapper` was a static method and we passed things to it, either `this` or the things it actually needs